.travis.yml

```diff
@@ -30,6 +54,14 @@ matrix:
         - os: osx
           language: generic
           env: CONDA=1 MPLBACKEND=Agg 
+        # OSX montepython_master
+        - os: osx
+          language: generic
+          env: MONTEPYTHONMASTER=1 MPLBACKEND=Agg
+        # OSX montepython_master conda
+        - os: osx
+          language: generic
+          env: MONTEPYTHONMASTER=1 CONDA=1 MPLBACKEND=Agg 
```

Comment:
Is this what you had in mind, Will, when you suggested testing for other platforms? Basically the tests work, however, on OSX `pip install pypolychord` fails. E.g. it works on my linux desktop but fails on my macbook. This seems more of an issue with `pypolychord` though and less with `anesthetic`, right?

Reply:
This is certainly good enough. I may submit a PR in the future neatening up some of this code, but for now it achieves what we want in a way that's generally consistent with the rest of the travis tests. Well done for getting it to work (the repeated pushes trying to get travis to work are part of the usual workflow of setting up continuous integration)

Reply:
Does anesthetic require ``pypolychord`` in any way? I can't see why we'd need it to be a dependency, and you are correct that pip install [does not currently work on OSX](https://github.com/PolyChord/PolyChordLite/issues/13) for PolyChord.

Reply:
no, `pypolychord` is not necessary here, which is why now montepython is installed without dependencies since commit [117d526](https://github.com/williamjameshandley/anesthetic/pull/14/commits/117d5261c5b430d90cf7c98fc81e1e18c2aa825a) .